Robot Code/src/org/usfirst/frc/team3316/robot/commands/climbing/PullUp.java

```diff
@@ -0,0 +1,53 @@
+package org.usfirst.frc.team3316.robot.commands.climbing;
+
+import org.usfirst.frc.team3316.robot.Robot;
+import org.usfirst.frc.team3316.robot.commands.DBugCommand;
+
+/**
+ * Command for climbing. Sets the climbing motors until we reach the target
+ * angle of the pot.
+ * 
+ * @author D-Bug
+ *
+ */
+public class PullUp extends DBugCommand {
+	private double speed, initAngle;
+	private boolean onTarget;
+
+	public PullUp()
+	{
+		requires(Robot.climbing);
```

Comment:
Maybe this command should require all of the robot's subsystems and stop the compressor for maximum power?